src/GraphQL.ApiTests/net50/GraphQL.approved.txt

```diff
@@ -2863,14 +2863,16 @@ namespace GraphQL.Types
     }
     public class ListGraphType : GraphQL.Types.GraphType, GraphQL.Types.IProvideResolvedType
     {
-        public ListGraphType(GraphQL.Types.IGraphType? type) { }
+        public ListGraphType(GraphQL.Types.IGraphType type) { }
         public GraphQL.Types.IGraphType? ResolvedType { get; set; }
         public virtual System.Type? Type { get; }
         public override string ToString() { }
     }
     public sealed class ListGraphType<[System.Diagnostics.CodeAnalysis.DynamicallyAccessedMembers(System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.PublicConstructors)]  T> : GraphQL.Types.ListGraphType
         where T : GraphQL.Types.IGraphType
     {
+        [System.Obsolete("This constructor is for internal use only; use ListGraphType(IGraphType? type) in" +
+            "stead.")]
         public ListGraphType() { }
```

Comment:
User code should never call this constructor.  The type is only a marking type for generics, used in scenarios such as:

```cs
Field<NonNullGraphType<MyGraphType>>("test");
```